sky/serve/infra_providers.py

```diff
@@ -622,10 +637,35 @@ def _probe_replica(info: ReplicaInfo) -> Tuple[str, bool]:
             if info.first_not_ready_time is None:
                 info.first_not_ready_time = time.time()
             if info.status_property.service_once_ready:
+                if self.use_spot:
+                    # Pull the actual cluster status from the cloud provider to
+                    # determine whether the cluster is preempted.
+                    (cluster_status, handle
```

Comment:
```suggestion
                    (cluster_status, _
```
It is an unused variable IIUC?

Reply:
Done, thanks.